tests/lit-tests/1963.ispc

```diff
@@ -0,0 +1,33 @@
+// This test does uint->float varying typecast and verifies that performance
+// warning is printed only for relevant cases.
+
+//; RUN: %{ispc} %s --emit-asm -o %t.sse2.s --nowrap --target=sse2 2>&1 | FileCheck %s -check-prefix=CHECK_SSE2_WARNING
```

Comment:
Do not use shot form of target, i.e. `sse2`, `avx2` - they are depricated.

Reply:
Switched to sse2-i32x4 and avx2-i32x8.